ush/python/pygfs/task/analysis.py

```diff
@@ -128,25 +127,21 @@ def get_obs_dict(self) -> Dict[str, Any]:
         return obs_dict
 
     @logit(logger)
-    def get_bias_dict(self) -> Dict[str, Any]:
-        """Compile a dictionary of observation files to copy
+    def get_bias(self) -> None:
```

Comment:
Sure, what do you have in mind?

Reply:
@RussTreadon-NOAA In #2805 , the ```Analysis``` class and thus all of ```analysis.py``` will become deprecated, and the ```get_bias_dict``` function will become a method of the```JEDI``` class.

That PR is essentially complete. I just have one minor comment from @aerorahul to resolve. If this PR is still in draft mode, maybe we should think towards making this change to the ```get_bias_dict`` either part of that PR, or merge our two PRs. What do you think?

Reply:
Thank you @DavidNew-NOAA for the update.  Since your PR is ready to go, let's get #2805 into g-w `develop` as soon as possoble.  We can then refactor the changes in this PR as needed.  This refactoring could be done in this PR or we abandon this PR and open a new one.

Reply:
@CoryMartin-NOAA , what do we know about the implementation of aircraft observation bias correction in JEDI?  Will the bias correction coefficients be carried around in a single netcdf or ascii file or will we need to collect aircraft bias correction files into a tarball like JEDI radiance bias correction?

Reply:
@RussTreadon-NOAA I'm on it. I have to merge and refactor (w/ respect the the JEDI class) the latest develop which has the new letkf observer code. Shouldn't be too long

Reply:
@RussTreadon-NOAA to my knowledge, all aircraft BC values should be in only one file, so `aircraft` would be analogous to `amsua_n19`.  The file would treat tail numbers like channels in terms of general structure.

Reply:
Thanks @CoryMartin-NOAA for this information.  Handling a single aircraft bias correction file is simple.  Do you think we can every combine the three radiance bias correction files into a single file?  This requires changes in JCSDA repos and so even if it is possible it won't happen in the near future.

Reply:
@RussTreadon-NOAA probably unlikely to happen anytime soon but plausible